src/cascadia/WindowsTerminal/IslandWindow.cpp

```diff
@@ -1717,5 +1717,13 @@ void IslandWindow::OpenSystemMenu(const std::optional<int> mouseX, const std::op
     }
 }
 
+void IslandWindow::AddToSystemMenu(const winrt::hstring& itemLabel, winrt::delegate<void()> callback)
+{
+}
+
+void IslandWindow::RemoveFromSystemMenu(const winrt::hstring& itemLabel)
+{
+}

```

Comment:
Ah, right, these clearly haven't been implemented yet. Makes sense why I wasn't seeing anything when checking this branch out 😅 

I'll leave this PR alone till it's marked "ready for review"